extensions/git-ui/src/main.ts

```diff
@@ -11,20 +11,21 @@ export async function deactivate(): Promise<any> {
 }
 
 export async function activate(context: ExtensionContext): Promise<void> {
-	context.subscriptions.push(commands.registerCommand('git.credential', async (data: any) => {
```

Comment:
It looks like they followed up with completely removing the git-ui extension 

https://github.com/microsoft/vscode/commit/40a6563f9b0ba6b8f29e0d0396a823252471f7ac#diff-20ef7739e8423f114899b767909a220c5dc4169d73a38196d7a2dbfac5166c4e

This is fine to keep as is - but might be worth looking into why they did that in case there's other reasons we might want to remove it now.

Reply:
does this have any impact on the git features in ADS? might have an impact for data-workspace features. @udeeshagautam FYI.